### DISCUSSION TITLE
Spotify playlist

### DISCUSSION THREAD

**Discussion by senkhaishi** [jagrosh/MusicBot#739] [Questions]
How to play spotify playlist, i tried making a Playlist but then when I type @prefix playlist (The name of the playlist) it fails to load

**Comment by YJSoft** [comment]
## Spotify
Playing from Spotify is not possible, for three main reasons:

1. This would require an additional API key to look up songs on Spotify, which would be confusing for some users and overall make the bot feel more complicated to use.
1. Streaming from Spotify isn't allowed, so the bot would have to "guess" and just play the closest match from YouTube, which is unreliable at best.
1. Discord already has native support for syncing up Spotify playlists between users, so there's no reason to not just use that (and get better audio anyway from Spotify instead of from a Discord voice channel).

https://github.com/jagrosh/MusicBot/wiki/Things-That-Won%27t-Be-Added
